logbook-core/src/main/java/org/zalando/logbook/BaseHttpMessage.java

```diff
@@ -20,8 +20,6 @@
  * #L%
  */
 
-import com.google.common.collect.Multimap;
-
 import java.nio.charset.Charset;
 
 public interface BaseHttpMessage {
```

Comment:
> Maybe the interface can be changed so that getHeader returns a Map<String, Collection<String>>, making the MultiMap completely an implementation detail?

The part of the contract where an non-existing key yields an empty collection doesn't work well when this is reduced to a traditional map, where it should be null.